### ISSUE TITLE
Add project milestones to Readme.md

### ISSUE THREAD

**Issue by NWoodsman** [Microsoft/Win2D#961]
It’s not apparent if this repo is going to be a viable Microsoft driven effort in the immediate future. Compared to repos like WinUI or Dotnet, which experience frequent updates, release targets, feature goals for releases, documentation of  meetings, etc. 

Is there a public interface for internal communication on this project? 

Can we add some Markdown content that describes the current state of the project, besides the one goal listed which is “complete the port from RT/ UWP to Winui3”?

**Comment by jschroedl** [comment]
Yes, please. I would like to know the level of commitments to the project before investing time using this tech.

**Comment by ChewWorker** [comment]
My team owns Win2D. I can say that we have internal and external partners relying on Win2D, and are committed to maintaining it. The main reason it doesn't get as many updates as WinUI is because the functionality in it is fairly stable. There have definitely been gaps in Documentation and functionality as we have been updating it for WinUI3 usage, but are continuing to close those based on customer needs.
